vm/ubuntu-vm.sh

```diff
@@ -130,11 +136,12 @@ function exit-script() {
 }
 
 function default_settings() {
+  RELEASE="${NOBLE}"
   VMID="$NEXTID"
   FORMAT=",efitype=4m"
   MACHINE=""
   DISK_CACHE=""
-  HN="ubuntu"
+  HN="ubuntu-${NOBLE}"
```

Comment:
What happens to hostname if you choose jammy in advanced settings? you have to manually fix it?